lib/panoptes-subject-uploader.iced

```diff
@@ -19,16 +19,30 @@ getMetadata = (rawData) ->
     metadata[key.trim()] = value.trim?() ? value
   metadata
 
-findImages = (searchDir, metadata) ->
+findImagesFiles = (searchDir, metadata) ->
   imageFiles = []
   for key, value of metadata
     imageFileName = value.match?(/([^\/]+\.(?:jpg|jpeg|gif|png|svg|mp4|txt))/i)?[1]
     if imageFileName?
       existingImageFile = glob.sync(path.resolve searchDir, imageFileName.replace /\W/g, '?')[0]
-      if existingImageFile? and  existingImageFile not in imageFiles
+      if existingImageFile? and existingImageFile not in imageFiles
         imageFiles.push existingImageFile
   imageFiles
 
+findImagesURLs = (metadata) ->
+  imageURLs = []
+  for key, value of metadata
+    httpsRegexPattern = new RegExp("^(http|https)://", "i")
```

Comment:
I did an actual test this morning and this regex is allowing both http and https. I got a http hosted image file uploaded. I think the regex needs to be `new Regex("^(https)://", "i")`